src/redturtle/prenotazioni/adapters/booker.py

```diff
@@ -101,28 +141,35 @@ def _create(self, data, duration=-1, force_gate=""):
             gate = available_gate
         else:
             gate = force_gate
+
+        fiscalcode = data.get("fiscalcode", "").upper()
+        user = api.user.get_current()
+
+        if not fiscalcode:
+            fiscalcode = (
+                user.getProperty("fiscalcode", "") or user.getId() or ""
+            ).upper()  # noqa
+
```

Comment:
aggiungere in params e verificare il limite solo se fiscalcode è valorizzato

Reply:
Done